sos/plugins/openstack_placement.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (C) 2019 Red Hat, Inc., Lee Yarwood <lyarwood@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class OpenStackPlacement(Plugin):
+    """OpenStack Placement
+    """
+    plugin_name = "openstack_placement"
+    profiles = ('openstack', 'openstack_controller',)
+
+    var_puppet_gen = "/var/lib/config-data/puppet-generated/placement"
+
+    def setup(self):
+
+        # collect commands output only if the openstack-placement-api service
+        # is running
+        service_status = self.get_command_output(
+            "systemctl status openstack-placement-api.service"
+        )
+
+        in_container = self.running_in_container()
+
+        if (service_status['status'] == 0) or in_container:
```

Comment:
Same as above. In fact you could remove `service_status` entirely here and replace it with something like

```python
if self.service_is_running('openstack-placement-api') or in_container:
```

Reply:
as above, thanks.